src/ucs/sys/string.c

```diff
@@ -133,11 +134,15 @@ const char* ucs_sockaddr_str(const struct sockaddr *sock_addr, char *str, size_t
     case AF_INET:
         addr_in = (struct sockaddr_in *) sock_addr;
         inet_ntop(AF_INET, &addr_in->sin_addr, str, max_size);
+        max_size -= strlen(str);
+        ucs_assert(max_size >= (1 + sizeof(in_port_t)));
```

Comment:
minor: it could be replaced by
ucs_assert(max_size > sizeof(in_port_t));

Reply:
Yes, i just thought this way it would be clear that the one is for the colon.

Reply:
i don't understand the purpose of this assertion.. sizeof(in_port_t) is always 2 (short) but %d will generate 1-5 characters when converting the number to a string.. and in any case snptintf() would make sure we don't overflow..